.eslintrc.js

```diff
@@ -81,6 +81,22 @@ module.exports = {
 						importNames: [ 'memoize' ],
 						message: 'Please use `memize` instead.',
 					},
+					{
+						name: 'lodash',
```

Comment:
It seems this overrides the above 'lodash` rule. See relevant [issue](https://github.com/eslint/eslint/issues/14220).

Reply:
Good catch @ntsekouras, thanks! Fixed in https://github.com/WordPress/gutenberg/pull/41651/commits/e41e003af2841e8fa7bd5d9e53a024bd071b0540. 

Since separate messages per import name are not supported at this time, the solution is to have one long message, which is not ideal but should do the job. Let me know what you think.